src/utils.test.js

```diff
@@ -214,6 +214,12 @@ describe('utils', () => {
     expect(utils.isStoreLogs()).toBe(true);
   });
 
+  test('isReuseHttpConnection', () => {
+    expect(utils.isReuseHttpConnection()).toBe(false);
+    process.env['LUMIGO_REUSE_HTTP_CONNECTION'] = 'TRUE';
+    expect(utils.isReuseHttpConnection()).toBe(true);
+  });
+
   test('setSendOnlyIfErrors', () => {
     expect(utils.isSendOnlyIfErrors()).toBe(false);
     utils.setSendOnlyIfErrors();
```

Comment:
Test idea: wrap a function with the tracer, run it twice, and make sure that only one connection was created.